sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/Clauses/ClauseBase.cs

```diff
@@ -1,17 +1,13 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System;
-using System.Collections.Generic;
-using System.Text;
-
 namespace Azure.DigitalTwins.Core.QueryBuilder
 {
     /// <summary>
     /// Clause objects make up an entire query. There are four different kinds of clauses, SELECT, FROM, JOIN and WHERE.
```

Comment:
nit comment: You are restricting the number of clauses that can implement the `ClauseBase` by naming them in the comments. What if someone adds a new clause? how do they know that they have to update the comments. 
We try to add doc comments that can outlive the initial implementation or design